Formula/ospac-gui.rb

```diff
@@ -0,0 +1,20 @@
+class OspacGui < Formula
+  desc "Open Source Podcast Audio Chain - GUI"
+  homepage "http://ospac.net/"
+  url "https://github.com/sritterbusch/ospac/archive/v1.0.tar.gz"
+  sha256 "793f0454cebe48dfa8364f26fa48469717954c31cb632a9c0b9dd24920d83f4c"
+
+  depends_on "libsndfile"
+  depends_on "fltk"
+  depends_on "ospac"
+
+  def install
+    system "make", "ospac-gui"
```

Comment:
Yes, when it's possible to use `inreplace` or a `patch do`, and then call `make install`, that is preferable to manual `blah.install`s.